src/agent/debuglet.ts

```diff
@@ -129,12 +129,12 @@ export class Debuglet extends EventEmitter {
   // was successful. It is returned by isReady.
   private breakpointFetched: Promise<void>|null;
   // breakpointFetchedResolve is the resolve function for breakpointFetched.
-  private breakpointFetchedResolve: (() => void);
+  private breakpointFetchedResolve: () => void;
   // debuggeeRegistered is a promise only to be resolved after debuggee
   // registration was successful.
   private debuggeeRegistered: Promise<void>;
   // debuggeeRegisteredResolve is the resolve function for debuggeeRegistered.
-  private debuggeeRegisteredResolve: (() => void);
+  private debuggeeRegisteredResolve: () => void;
```

Comment:
Nit - Given the similarity of `breakpointFetched` and `debuggeeRegistered` I would still like to see sort of abstraction, especially because you already wrote one in another PR.

Reply:
Done.